test_files/decorator/external.js

```diff
@@ -17,3 +17,5 @@ exports.AClassWithGenerics = AClassWithGenerics;
  */
 function AType() { }
 exports.AType = AType;
+function AType_tsickle_Closure_declarations() {
+}
```

Comment:
I saw that, but I figured that this shouldn't hurt assuming rollup always figures out correctly that this unused (right?), and I'd rather have the code less branchy. Also doesn't add too much code in development mode.